suite-common/token-definitions/scripts/constants/index.ts

```diff
@@ -1,6 +1,6 @@
 import { join, resolve } from 'path';
 
-import { VERSION } from '../../src/constants';
+import { VERSION } from '../../src/tokenDefinitionsConstants';
```

Comment:
why change of names of those files? it does not make sense to rename contracts to tokenDefinitionsConstants in token definitions package

Reply:
I did this because in other packages that are also inside `suite-common` like `analytics`, `device-authenticity`, `message-system` it's exactly like this so I thought it sort of convention to name actions, reducer etc. files like this.

Reply:
Oka. Then it should not be part of this PR or it should be at least in separate commit

Reply:
d4986939dc9f00c0ea53dcc6ba8ec07daff2bc18